app/src/main/java/org/gdg/frisbee/android/onboarding/FirstStartStep1Fragment.java

```diff
@@ -53,21 +60,39 @@
 
     private static final String ARG_SELECTED_CHAPTER = "selected_chapter";
     @Bind(R.id.chapter_spinner)
-    Spinner mChapterSpinner;
+    AutoCompleteSpinnerView autoCompleteSpinnerView;
     @Bind(R.id.confirm)
-    Button mConfirm;
+    Button mConfirmButton;
     @Bind(R.id.viewSwitcher)
     ViewSwitcher mLoadSwitcher;
-    private ChapterAdapter mSpinnerAdapter;
+    private ChapterAdapter mChapterForCityName;
```

Comment:
Why do they have different layouts? I think having the list in 2 adapters doesn't make sense. 

What about this? We can make the list always sorted by location and then we can have filter.